templates/tweets/scripts.html

```diff
@@ -0,0 +1,52 @@
+<script type="text/javascript">
+  const getCookie = name => {
+    if (document.cookie && document.cookie !== "") {
+      for (const cookie of document.cookie.split(";")) {
+        const [key, value] = cookie.trim().split("=");
+        if (key === name) {
+          return decodeURIComponent(value);
+        }
+      }
+    }
+  };
+  const csrftoken = getCookie('csrftoken');
+
+
+  const changeStyles = jsonResponse => {
+    const selector = document.getElementsByName(jsonResponse.tweet_id);
+    if (jsonResponse.liked) {
+      console.log("liked")
+      selector.innerHTML = "<i class='fas fa-lg fa-heart like-red'></i>";
+    } else {
+      console.log("not liked")
+      selector.innerHTML = "<i class='far fa-lg fa-heart'></i>";
+    }
+    document.getElementsByName(jsonResponse.tweet_id + "-count").textContent = jsonResponse.count;
```

Comment:
こちらも同様です。NodeList直下にはtextContentメソッドはないので参照できないです。
```suggestion
    document.getElementsByName(jsonResponse.tweet_id + "-count")[0].textContent = jsonResponse.count;
```
とすればいけます

Reply:
修正しました。